AspNetCore.SassCompiler/SassCompiler.cs

```diff
@@ -43,9 +43,9 @@ public async Task CompileAsync(string[] args)
 
         process.Start();
         ChildProcessTracker.AddProcess(process);
-        await process.StandardOutput.BaseStream.CopyToAsync(Stream.Null);
-        await process.StandardError.BaseStream.CopyToAsync(errorOutput);
-        await process.WaitForExitAsync();
+        Task outputTask = process.StandardOutput.BaseStream.CopyToAsync(Stream.Null);
```

Comment:
We mainly use `var` for all variables in this repo, can you change this to use `var` as well?